src/InjectionContainer/InjectionContainer.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\View\InjectionContainer;
+
+use Psr\Container\ContainerInterface;
+
+final class InjectionContainer implements InjectionContainerInterface
```

Comment:
Looks like a dummy class that does nothing. Can we avoid it?

Reply:
We can use directly PSR container instead of it. But there are disadvantages:

1) unnecessary method `has()`
2) method `get()` returns `mixed`, so we should check that it is object and throw exception if no.